src/sentry/static/sentry/app/components/group/suggestedOwners.jsx

```diff
@@ -181,12 +181,13 @@ const SuggestedOwners = createReactClass({
                   commits={owner.commits}
                   containerClassName="avatar-grid-item"
                 >
-                  <ActorAvatar
-                    style={{cursor: 'pointer'}}
-                    hasTooltip={false}
-                    actor={owner.actor}
-                    onClick={() => this.assignToActor(owner)}
-                  />
+                  <span onClick={() => this.assign(owner.actor)}>
```

Comment:
Is this equivalent? Previously it was calling `assignToActor(owner)`

Reply:
Yeah, it was wrong. There was `assignToActor` and `assignToUser`, the later never being used. and `assingToActor` only being used for team assignment